src/third-party/PhishingController.ts

```diff
@@ -133,30 +136,44 @@ export class PhishingController extends BaseController<
     };
     this.detector = new PhishingDetector(this.defaultState.phishing);
     this.initialize();
-    this.poll();
   }
 
   /**
-   * Starts a new polling interval.
+   * Set the interval at which the phishing list will be refetched. Fetching will only occur on the next call to test/bypass. For immediate update to the phishing list, call updatePhishingLists directly.
+   * last this.lastFetched.
    *
-   * @param interval - Polling interval used to fetch new approval lists.
+   * @param interval - the new interval, in ms
    */
-  async poll(interval?: number): Promise<void> {
-    interval && this.configure({ interval }, false, false);
-    this.handle && clearTimeout(this.handle);
-    await safelyExecute(() => this.updatePhishingLists());
-    this.handle = setTimeout(() => {
-      this.poll(this.config.interval);
-    }, this.config.interval);
+  setRefreshInterval(interval: number) {
```

Comment:
If the `refreshInterval` can already be set via `configure`, then what do you think about having that be the only way to do this? I just think if we have a mechanism for setting configuration options, then we should make use of it instead of trying to paper it.

Reply:
Would using configure make it more difficult in the future when we want to upgrade the PhishingController to use BaseControllerV2? (afaik the v2 doesnt use .configure anymore, correct?)

Reply:
You're right, it doesn't. That's a good point. This is good then!